sdk/python/feast/expediagroup/vectordb/milvus_online_store.py

```diff
@@ -296,3 +305,96 @@ def _format_data_for_milvus(self, feast_data):
 
         transformed_data = [list(item) for item in zip(*milvus_data)]
         return transformed_data
+
+    def _convert_milvus_result_to_feast_type(
+        self, milvus_result, collection, features_to_request
+    ):
+        """
+        Convert Milvus result to Feast types.
+
+        Parameters:
+        milvus_result (List[Dict[str, any]]): Milvus query result.
+        collection (Collection): Milvus collection schema.
+        features_to_request (List[str]): Features to request from Milvus.
+
+        Returns:
+        List[Dict[str, ValueProto]]: Processed data with Feast types.
+        """
+
+        # Here we are constructing the feature list to request from Milvus with their relevant types
+
+        features_with_types = list(tuple())
+        for field in collection.schema.fields:
+            if field.name in features_to_request:
+                features_with_types.append(
+                    (field.name, self._milvus_to_feast_type(field.dtype))
+                )
+
+        feast_type_result = []
+        value_type_actions = {
```

Comment:
can you add the other data types as well? boolean, int32, float, etc

Reply:
reading from a value proto is done multiple times in the code and should be wrapped into a separate method